download/en-us/download.md

```diff
@@ -7,8 +7,8 @@ Use the links below to download the Apache DolphinScheduler from one of our mirr
 ## DolphinScheduler
 | Date | Version| | Downloads |
 |:---:|:--:|:--:|:--:|
-| Mar. 7th, 2022 | 2.0.5 | Total Source Code| [[src]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-src.tar.gz.sha512)|
-| | | Total Binary Distribution| [[bin]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-bin.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-bin.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-bin.tar.gz.sha512)|
+| Mar. 7th, 2022 | 2.0.5 | Total Source Code| [[src]](https://www.apache.org/dyn/closer.lua/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-src.tar.gz) [[asc]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-src.tar.gz.asc) [[sha512]](https://downloads.apache.org/dolphinscheduler/2.0.5/apache-dolphinscheduler-2.0.5-src.tar.gz.sha512)|
```

Comment:
Can we use `https://downloads.apache.org/dolphinscheduler/2.0.5/` directly?

Reply:
It's better to 'Use the links below to download the Apache DolphinScheduler from one of our mirr'